pkg/cli/loader_flag.go

```diff
@@ -17,6 +18,15 @@ func (*FlagLoader) Load(args []string, cmd *cli.Command) (bool, error) {
 		return false, nil
 	}
 
+	if cmd.Name == "healthcheck" {
+		for _, a := range args {
+			if strings.HasPrefix(a, "--url") {
+				log.Print("Skipping flag load for healthcheck (url provided)")
+				return false, nil
+			}
+		}
+	}
+
```

Comment:
Avoiding parsing of the static configuration seems superfluous, just ignoring it in the healthcheck command is enough.
```suggestion
```